bin/fin

```diff
@@ -1474,8 +1475,7 @@ show_help_project() {
 	echo
 	printh "status" "List project services (alias: fin ps)"
 	printh "restart" "Restart project services (alias: fin restart)"
-	printh "reset [service]" "Recreate all or specified project services, their containers and named volumes"
-	printh "" "Changes to home directory in \`cli\` are preserved."
+	printh "reset [service]" "Recreate all or specified project services, their containers and volumes"
```

Comment:
The wording about "Changes to home directory" needs to be returned. Please see conversation for details.